### ISSUE TITLE
Resizable separator for Browse Collection section?

### ISSUE THREAD

**Issue by Searinox** [zlatinb/muwire#92]
The current collection layout is 1/3 of the window height for the list, 1/3 for the comment and only 1/3 for the folder tree itself. Even at 1080p that tree browsing space is quite narrow and it would help a lot with navigation if the user could resize these to get a better look at the folder tree.

**Comment by zlatinb** [comment]
I agree the Browse Collections view doesn't look good at all.  I want to redesign it completely.  Just making the two separators resizable is going to help but it still won't look good.  So let's leave this open for ideas for a new design.

**Comment by Searinox** [comment]
Okay. Personally I have no other ideas to suggest though lol.

**Comment by zlatinb** [comment]
Well one thing that would make it more consistent with the other views is if there is only two panels - top and bottom.  The top would be the list of collections, the bottom would be the tree/table of files.  If the collection has a description, a "View description" button can be added.. or now that I know how to use tooltips it would be possible to show parts of the description in a tooltip when hovering over the collection name.
